cpu/stm32/include/clk/cfg_clock_common_fx_gx_mp1_cx.h

```diff
@@ -13,15 +13,15 @@
  * @{
  *
  * @file
- * @brief       Base STM32Fx/Gx/MP1 clock configuration
+ * @brief       Base STM32Fx/Gx/MP1/Cx clock configuration
```

Comment:
I don't think there was much thought put into the order of those lists, they were just extended as needed.

Since there is no other Cx family than C0, I wouldn't write Cx as that would imply support for future families that we don't yet support.

Reply:
"cx" replaced with "c0" and file renamed